packages/ckeditor5-special-characters/docs/features/special-characters.md

```diff
@@ -96,6 +96,10 @@ Below you can see a demo based on the example shown above. Use the special chara
 
 {@snippet features/special-characters-new-category}
 
+<info-box>
+	You may also check out the ready-to-use {@link features/emoji emoji feature} in order to quickly insert desired emoji into the document.
+</info-box>
```

Comment:
I decided to just add a cross-link note to the special characters feature guide, where we teach how to add a custom "Emoji" category instead of rewriting it.

Adding the "Emoji" category is just an example of extending the special characters feature, so I think there is no need to change the content of this paragraph, even if we already have a full-size emoji feature. Just like we describe diesel locos in some example document and we do not require that users should also write about locos in their documents.